### ISSUE TITLE
AspNetClaims updating a claim from a role 

### ISSUE THREAD

**Issue by jvelezc** [aspnet/Identity#811]
The fact that an extension method is needed to remove claims in either the aspnetuserclaim and aspnetuserroleclaims is not the best way to interact with the system. It would be best if 
just like there is 
pseudo code
RoleManager<ApplicationRole> _roleManager = 
Exist today
_roleManager.AddClaimAsync
_roleManager.RemoveClaimAsync
My proposal: 
There should also be UpdateClaimAsync

http://stackoverflow.com/questions/24587414/how-to-update-a-claim-in-asp-net-identity

Can I do a PR?

**Comment by divega** [comment]
@haok didn't we do this already? maybe only for user and not for role?

**Comment by HaoK** [comment]
Yeah someone added ReplaceClaim for UserManager, but we didn't add anything for role claims.

@jvelezc feel free to submit a PR

https://github.com/aspnet/Identity/blob/dev/src/Microsoft.AspNetCore.Identity/UserManager.cs#L978

**Comment by blowdart** [comment]
Bumping to post RTM

**Comment by HaoK** [comment]
Easy fix to make

**Comment by HaoK** [comment]
This is breaking change unfortunately as we need to add ReplaceClaim to IRoleClaimStore, its a fairly straightforward change, but needs to wait for now
